NeuroID/src/main/java/com/neuroid/tracker/service/NIDServiceTracker.kt

```diff
@@ -140,7 +141,8 @@ object NIDServiceTracker {
             put("pageTag", screenActivityName)
             put("pageId", rndmId)
             put("tabId", rndmId)
-            put("responseId", sharedDefaults.generateUniqueHexId())
+            put("responseId", sharedDefaults.generateUniqueHexId(
+               UUID.randomUUID().toString(), System.currentTimeMillis()))
```

Comment:
Rather than change the signature for the `generateUniqueHexId` and have to pass in params every time, why don't we just move these two into the function itself so nothing is required and the function does all the imports/work?